statsdbinterface/database/extmodels.py

```diff
@@ -2,6 +2,7 @@
 from werkzeug.exceptions import NotFound
 from .core import db
 from .models import Game, GamePlayer, GameServer, GameWeapon
+from .models import Mutator as GMut, Mode as GMode
```

Comment:
It is bad style to split up imports from the same file into multiple lines. Also, why do you rename the classes here? One shouldn't do that as well.